PicaSWB.js

```diff
@@ -220,21 +212,11 @@ function doExport() {
 			writeLine("4070", volumeyearissuepage);
 		}
 		
-		//URL --> 4085
+		//URL --> 4085 nur bei Katalogisierung nach "Oox" im Feld 0500 "Oox"
 		if (item.url) {
 			writeLine("4085", item.url + "$xH");
```

Comment:
Okay, aber dann lass uns lieber hier nach dem Code "Oox" suchen und in die `if`-Bedingung hineinnehmen, dann muss man nicht verschiedene Versionen des Skriptes pflegen.